spec/features/bars/user_deletes_bar_spec.rb

```diff
@@ -2,8 +2,8 @@
 
 feature "authenticated user deletes an existing bar" do
   scenario "successfully deletes bar" do
-    User.create(email: "mckelvey.matt@gmail.com", password: "12345678")
-    Bar.create(name: "Happy Hour Lasagna", description: "Where are we at?")
+    user1 = User.create(email: "a2@gmail.com", password: "12345678")
```

Comment:
For the next iteration, follow the pattern that you have in `user_edits_bar_spec.rb`

```
let(:new_user) { User.create(email: "a3@gmail.com", password: "12345678") }
let(:bar1) { Bar.create(name: "Happy Hour Lasagna", description: "Where are we at?", user: new_user) }
```